packages/relay/src/lib/services/debugService/index.ts

```diff
@@ -0,0 +1,261 @@
+   * @example
+   * const result = await debug_traceTransaction('0x123abc', TracerType.CallTracer, {"tracerConfig": {"onlyTopCall": false}}, some request id);
+   */
+  async debug_traceTransaction(
+    transactionHash: string,
+    tracer: TracerType,
+    tracerConfig: object,
+    requestIdPrefix?: string,
+  ): Promise<any> {
+    this.logger.trace(`${requestIdPrefix} debug_traceTransaction(${transactionHash})`);
+    try {
+      DebugService.requireDebugAPIEnabled();
+      if (tracer === TracerType.CallTracer) {
+        return await this.callTracer(transactionHash, tracerConfig, requestIdPrefix);
+      } else if (tracer === TracerType.OpcodeLogger) {
+        throw predefined.UNSUPPORTED_METHOD;
+      }
+    } catch (e) {
+      throw this.common.genericErrorHandler(e);
+    }
+  }
+
+  /**
+   * Formats the result from the actions endpoint to the expected response
+   *
+   * @async
+   * @param {any} result - The response from the actions endpoint.
+   * @param {string} requestIdPrefix - The request prefix id.
+   * @returns {Promise<[] | any>} The formatted actions response in an array.
+   */
+  async formatActionsResult(result: any, requestIdPrefix?: string): Promise<[] | any> {
+    return await Promise.all(
+      result.map(async (action, index) => {
+        const { resolvedFrom, resolvedTo } = await this.resolveMultipleAddresses(
+          action.from,
+          action.to,
+          requestIdPrefix,
+        );
+
+        // The actions endpoint does not return input and output for the calls so we get them from another endpoint
+        // The first one is excluded because we take its input and output from the contracts/results/{transactionIdOrHash} endpoint
+        const getContract =
+          index !== 0 && action.call_operation_type === 'CREATE'
+            ? await this.mirrorNodeClient.getContract(action.to, requestIdPrefix)
+            : undefined;
+
+        return {
+          type: action.call_operation_type,
+          from: resolvedFrom,
+          to: resolvedTo,
+          gas: numberTo0x(action.gas),
+          gasUsed: numberTo0x(action.gas_used),
+          input: getContract?.bytecode ?? action.input,
+          output: getContract?.runtime_bytecode ?? action.result_data,
+          value: numberTo0x(action.value),
+        };
+      }),
+    );
+  }
+
+  /**
+   * Returns an address' evm equivalence.
+   *
+   * @async
+   * @param {string} address - The address to be resolved.
+   * @param {[string]} types - The possible types of the address.
+   * @param {string} requestIdPrefix - The request prefix id.
+   * @returns {Promise<string>} The address returned as an EVM address.
+   */
+  async resolveAddress(
+    address: string,
+    types = [constants.TYPE_CONTRACT, constants.TYPE_TOKEN, constants.TYPE_ACCOUNT],
+    requestIdPrefix?: string,
+  ): Promise<string> {
+    const entity = await this.mirrorNodeClient.resolveEntityType(
+      address,
+      types,
+      'debug_traceTransaction',
```

Comment:
could this be a constant similar to `eth.ts` endpoints/methods?  

https://github.com/hashgraph/hedera-json-rpc-relay/blob/5e38be160980685074b2574950e8e34f89bf0577/packages/relay/src/lib/eth.ts#L96C6-L110

Reply:
Yes :)